integration/network/macvlan/macvlan_driver.bats

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../../lib/common.bash"
+
+# Image for macvlan testing
+IMAGE="debian"
+# Network name
+NETWORK_NAME="macvlan1"
+# Network driver
+NETWORK_DRIVER="macvlan"
+# Containers names
+FIRST_CONTAINER_NAME="containerA"
+SECOND_CONTAINER_NAME="containerB"
+# Number of packets
+PACKET_NUMBER="5"
+
+setup () {
+	clean_env
+
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "ping container with macvlan driver" {
+	# Create network
+	docker network create -d ${NETWORK_DRIVER} ${NETWORK_NAME}
+
+	# Run the first container
+	docker run -d --runtime=${RUNTIME} --network=${NETWORK_NAME} --name=${FIRST_CONTAINER_NAME} ${IMAGE} sh -c "tail -f /dev/null"
+
+	# Verify ip address
+	ip_address=$(docker exec ${FIRST_CONTAINER_NAME} ip a | awk '/inet / {print $2}' | cut -d'/' -f1 | tail -1)
```

Comment:
Could you also verify this against maybe a docker network inspect as well? Just curious, as in other tests we extract the container IP using the docker inspect tools on the host side.